### DISCUSSION TITLE
Allow backing up all databases

### DISCUSSION THREAD

**Discussion by nahtnam** [coollabsio/coolify#3358] [Improvement Requests]
Currently in the database backups option, you must define every database youd like to back up. It would be nice if we could insert `*` to back up every database that exists in that postgres instance

**Comment by sneko** [comment]
I agree, for a shared database it's complicated to keep in sync what databases have been created and this list.

I guess if the username filled into Coolify database is still the superuser it will be able to see other databases.

@andrasbacsai any thought on this feature request?

**Comment by jere-co** [comment]
This would also be very convenient with some services. For example EdgeDB creates several databases with random names, some of which might even update as the project gets updated in the future.

**Comment by jere-co** [comment]
In a live stream @andrasbacsai added this option - so should be in production soon!

**Comment by andrasbacsai** [comment]
It is in the latest version. Works with Mysql, Mariadb and Postgresql.
